tests/integration/mixer/check_test.go

```diff
@@ -71,7 +71,7 @@ func TestCheck_Allow(t *testing.T) {
 				}
 
 				return nil
-			}, retry.Timeout(time.Second*40))
+			}, retry.Delay(30*time.Second), retry.Timeout(180*time.Second))
```

Comment:
30s between waits seems pretty high. Any reason to not check every second?

Reply:
I have seen in tests that 30s is needed for the rules to sync in general.. changing it to be 15s.. so that if its done sooner, we don't wait for this long